src/sage/matrix/matrix2.pyx

```diff
@@ -901,7 +901,6 @@ cdef class Matrix(Matrix1):
             sage: v = matrix.identity(QQ, 500).solve_right(vector(QQ, [1]*500), extend=False)  # <1s
             sage: matrix.identity(QQ, 500).hermite_form()  # not tested (slow)
             sage: v = (matrix.identity(ZZ, 500)*2).solve_right(vector(ZZ, [2]*500), extend=False)  # <1s
-            sage: matrix.identity(ZZ, 500).hermite_form()  # not tested (slow)
```

Comment:
This line was deleted because (surprisingly) it's actually fast. Only hermite_form of 2 × Id or something similar is slow.